README.md

```diff
@@ -455,6 +455,7 @@ This repository contains a curated list of awesome open source libraries that wi
 
 ## Industry Strength NLP
 * [AdaptNLP](https://github.com/Novetta/adaptnlp) ![](https://img.shields.io/github/stars/Novetta/adaptnlp.svg?style=social) - Built atop Zalando Research's Flair and Hugging Face's Transformers library, AdaptNLP provides Machine Learning Researchers and Scientists a modular and adaptive approach to a variety of NLP tasks with an Easy API for training, inference, and deploying NLP-based microservices.
+* [Argilla](https://github.com/argilla-io/argilla) ![](https://img.shields.io/github/stars/argilla-io/argilla.svg?style=social) - Argilla helps domain experts and data teams to build better NLP datasets in less time.
```

Comment:
It seems Argilla is more relevant for data labelling

Reply:
That is a good question, @axsaucedo I am thinking about the current solo categorization in the current structure of our list since we are not allowed to attribute more than one label to a single tool as indicated by https://github.com/EthicalML/awesome-production-machine-learning/issues/383.

As far as I understand, if a tool is designed for data labeling, it should be universally used for data labeling, which means, ideally practitioners, can use it to label their data regardless of their working domain. 

On the other hand, if a tool only caters to a specific domain, such as the NLP dataset. It is a domain tool that is built exclusively for NLP practitioners (since other practitioners cannot use it for other labeling purpoess). Then we should categorize it as an NLP tools rather than a general data labeling tool. What's your idea? @axsaucedo

Reply:
Looking at the repo it does seem better fit for data labelling than general NLP as invovles dataset mgmt as well

Reply:
> Looking at the repo it does seem better fit for data labelling than general NLP as invovles dataset mgmt as well

I know this tool has the functionalies of dataset management but it is exclusively focused on NLP dataset, for example, we cannot use this tool to create/manage/visualize other modal dataset such as images, audio, code, etc... @axsaucedo

Reply:
IMO that's ok, what you suggest is just multimodal data labelling, it's ok for datalabelling being single modality

Reply:
> IMO that's ok, what you suggest is just multimodal data labelling, it's ok for datalabelling being single modality

I think it is better to discuss it over here: https://github.com/EthicalML/awesome-production-machine-learning/issues/387
I found a lot of times this kind issue make a dilemma for the contributors to decide which categories to choose for a new tool. @axsaucedo

Reply:
For sure, if you move it to data labelling we can continue discussing in issue, alternative option is to remove from PR

Reply:
I think it is better to know what makes things right and then do the right thing, rather than make a detour. I have proposed some issues but nobody discussed them at all. If so,  why should I make the issues? @axsaucedo